tests/clients/test_mqtt_client.py

```diff
@@ -219,9 +219,9 @@ def on_command(client, obj, msg):
         httpc.iota.post_group(service_group=self.service_group_json, update=True)
         httpc.iota.post_device(device=self.device_json, update=True)
 
-        mqtt_broker_url = urlparse(settings.MQTT_BROKER_URL)
+        mqtt_broker_url: AnyUrl = settings.MQTT_BROKER_URL
```

Comment:
try to define an MQT-URL-Type

Reply:
Sure,  there is a custom `AnyMqttUrl` that can be used